### ISSUE TITLE
Cannot support via Sponsor

### ISSUE THREAD

**Issue by 4lch4** [Nekos-life/nekos-dot-life#14]
**Describe the bug**
When attempting to support the project via the Sponsor button, I get an error that one of the users provided in the FUNDING.yaml isn't registered for sponsors.

**To Reproduce**
Click the Sponsor button at the top of the repo.

**Expected behavior**
I've never sponsored a repo before so I'm not entirely sure what it _should_ look like.

**Desktop (please complete the following information):**
 - Node version [node -v]
N/A

**Comment by TheCyberRonin** [comment]
Yeah, that's of no fault to you. I've yet to be accepted into the GH Sponsors program 😭
